.github/workflows/ci-cd.yml

```
@@ -85,11 +85,35 @@ jobs:
     needs: [lint, typecheck, backend-tests]
     if: github.ref == 'refs/heads/main'
     runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
 
+      - name: Setup PNPM
+        uses: pnpm/action-setup@v2
+        with:
+          version: 10.6.3
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)